assignment/assignment-impl/impl/src/java/org/sakaiproject/assignment/impl/BaseAssignmentService.java

```diff
@@ -11742,6 +11707,62 @@ else if (typeOfGrade == Assignment.CHECK_GRADE_TYPE) {
 				}
 			}
 		}
+		
+		
+		public String getGradeForUserInGradeBook(String userId)
+		{
+			return this.getGradeFromGradebook(userId);
+		}
+		
+		protected String getGradeFromGradebook(String userId)
+		{
+			String rv =null;
+			if (userId == null) 
+			{
+				userId = m_submitterId;
+			}
+			Assignment m = getAssignment();
+			String gAssignmentName = StringUtils.trimToNull(m.getProperties().getProperty(AssignmentService.PROP_ASSIGNMENT_ASSOCIATE_GRADEBOOK_ASSIGNMENT));
+			if (gAssignmentName != null)
+			{
+				GradebookService g = (GradebookService)  ComponentManager.get("org.sakaiproject.service.gradebook.GradebookService");
+				String gradebookUid = m.getContext();
+
+				// return student score from Gradebook
```

Comment:
I did not know this method. I just wanted to change as little as possible. I need that the new method returns the grades internationalized so that there is no problem between decimal separtors ("." or ","). If thats not possible, I'll need to treat the grades to avoid i18n problems. @steveswinsburg any thoughts on that?